src/localization/locales/en-US.json

```diff
@@ -498,6 +498,7 @@
         "delete_label": "Delete “{{name}}”",
         "download_label": "Download “{{name}}”",
         "invalid_extension": "The file extension ({{file.resourceType}}) does not match the file’s contents",
+        "invalid_extension_zip": "[TODO] Invalid shapefile",
```

Comment:
issue: this string does not appear to be used anywhere. should it be removed?

Reply:
@paulschreiber the extension comes from the context of the error message from the api, so we can customize it like that, in this case it is useful because there is a validation for zip shapefiles, if we remove this it will give the user a generic `The file extension (.zip) does not match the file’s contents` and it is not true since it can be a valid zip but not a valid zip shapefile